XMonad/Actions/WithAll.hs

```diff
@@ -50,3 +51,7 @@ withAll f = withWindowSet $ \ws -> let all' = integrate' . stack . workspace . c
 -- | Kill all the windows on the current workspace.
 killAll :: X()
 killAll = withAll killWindow
+
+-- | Kill all the unfocused windows on the current workspace.
```

Comment:
Now that `WithAll` contains functions that do not act on all windows, it might be nice to mention this somewhere in the documentation of the module, as well as X.D.Extending

Reply:
What would you like in the module documentation?

```
-- Provides functions for performing a given action on all windows and
-- all unfocused windows of the current workspace.
```

Reply:
We could also now add:

```
sinkOthers :: X ()
sinkOthers = withUnfocused sink
```

Reply:
> What would you like in the module documentation?

Perhaps something about "all or only certain groups of windows" 

> rename this module something like XMonad.Actions.Grouped?

It would probably be for the best, but renaming modules and keeping their old version around is always a bit of a pain :/

Reply:
> Perhaps something about "all or only certain groups of windows"

how does this look?
  
> It would probably be for the best, but renaming modules and keeping their old version around is always a bit of a pain :/

We could have `Xmonad.Actions.Grouped` with everything and then this module can re-export the old functions?

Reply:
> how does this look?

Looks fine to me; thanks!

> We could have `Xmonad.Actions.Grouped` with everything and then this module can re-export the old functions?

Indeed; but we'd probably still want to deprecate `WithAll` in that case.  I'm not sure it's quite warranted right now with just one additional function, but maybe as things evolve we'll eventually be forced to do this

Reply:
That  makes sense. Maybe we can put together a list of additional grouped actions we would like? `sinkOthers` is the first obvious one I can think of, but it will require a variation of `withUnfocused` like `withAll'`

Reply:
I have to say I've never felt the need for acting on so many windows at once, so naturally I can't think of any :)  This may or may not be the PR "feature creeping", so perhaps it's better to get this small change merged now and think of adding an X.A.Grouped module in another step; what do you think?

Reply:
Yeah that makes sense to me. Lets get this one merged and I'll write a new issue to discuss changing the module